NotesBackEnd/src/Notes.Contracts/Services/Notes/IGetNoteService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Threading.Tasks;
+using Notes.Contracts.Model;
+
+namespace Notes.Contracts.Services.Notes
+{
+    public interface IGetNoteService

```

Comment:
This is not a good name - it does not even sound English 😉 
Look the word `get` in a dictionary or find another way to get rid of the noun at the place of an adjective.

Reply:
Same applies to the `IAddNoteService`…

Reply:
Same applies to the `IUpdateNoteService`…

Reply:
I'm starting to re-think whether forbidding the suffix `Service` would actually help. I was always afraid that you (~ any of you) would come with names like `NoteUpdater` or `NewNoteCreator` 😆

Reply:
I made same renamings but I am not sure if it is better now. If not, I will fix it after consultation with you.